plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -43,9 +41,20 @@ export class TeamManager {
     constructor(db: DB, serverConfig: PluginsServerConfig, statsd?: StatsD) {
         this.db = db
         this.statsd = statsd
-        this.teamCache = new Map()
-        this.eventDefinitionsCache = new Map()
         this.lruCacheSize = serverConfig.EVENT_PROPERTY_LRU_SIZE
+
+        this.teamCache = new LRU({
+            max: 10000,
+            maxAge: 2 * ONE_MINUTE,
+            // being explicit about the fact that we want to update
+            // the team cache every 2min, irrespective of the last access
+            updateAgeOnGet: false,
+        })
+        this.eventDefinitionsCache = new LRU({
```

Comment:
Should there be an equivalent change for `eventDefinitionsCache` e.g. to use not use `getByAge`?

Reply:
Ah looks like it already uses .get anyway

Reply:
it already doesn't use `getByAge`

Reply:
oh seems my comment was sent late? :bad-internet: